apps/web/app/services/client/api/task-sizes.ts

```diff
@@ -21,6 +22,23 @@ export function deleteTaskSizesAPI(id: string) {
 	return api.delete<DeleteResponse>(`/task-sizes/${id}`);
 }
 
-export function getTaskSizesList(tenantId: string, organizationId: string, activeTeamId: string | null) {
-	return api.get(`/task-sizes?tenantId=${tenantId}&organizationId=${organizationId}&activeTeamId=${activeTeamId}`);
+export async function getTaskSizesList(tenantId: string, organizationId: string, activeTeamId: string | null) {
+	const endpoint = `/task-sizes?tenantId=${tenantId}&organizationId=${organizationId}&organizationTeamId=${activeTeamId}`;
+
+	// API call direct to Backend Server
+	if (process.env.NEXT_PUBLIC_GAUZY_API_SERVER_URL) {
```

Comment:
@badalkhatri0924 can we put all that code `if (process.env...)...` inside of `api.get(...)` method? So that we don't duplicate SAME in ALL methods like that? We can also add one more parameter "isDirect" to the api.get signature like this:

```
return api.get(endpoint, true) 
```

So that methods that should work with direct way will do it and other methods where we still have `return api.get(endpoint) ` will not need to do it? etc.

Basically my concern is that all methods like `getTaskSizesList` will not have lots of additional not relevant code, while we can use either "factory" pattern or just "abstract" direct / not direct inside api.get method itself... Btw, same for all other methods etc.

Reply:
@evereq please review this again.